tabbycat/participants/models.py

```diff
@@ -9,6 +9,8 @@
 from django.utils.functional import cached_property
 from django.utils.translation import gettext_lazy as _
 
+import privateurls
```

Comment:
Rather import just `populate_url_keys` rather than the whole module.

Reply:
Then you run into circular imports, for now I've tried this as possibly a better alternative...